functional/FunctionalTest.java

```diff
@@ -3558,9 +3559,8 @@ public static void main(String[] args) throws Exception {
     } else {
       kmsKeyName = System.getenv("MINIO_JAVA_TEST_KMS_KEY_NAME");
       if (kmsKeyName == null) {
-        kmsKeyName = System.getenv("MINT_KEY_ID");
```

Comment:
Are you sure the env is changed from `"MINT_KEY_ID"` to `"MINT_KMS_KEY_NAME"`?

Reply:
there is no MINT_KEY_ID env in mint @balamurugana - I just added this to make sure we keep it the same as MINIO_JAVA_TEST_KMS_KEY_NAME

Reply:
`MINT_KEY_ID` was introduced in https://github.com/minio/minio-java/pull/694 and available since last two years. Do you say it was a mistake in the PR?

Reply:
Yes it is not at all present in mint, it has been long removed.